docs/2.guide/5.best-practices/performance.md

```diff
@@ -0,0 +1,346 @@
+---
+navigation.title: 'Nuxt Performance'
+title: Nuxt performance
+description: Best practices for improving performance of Nuxt apps
+---
+
+# Performance
+
+Nuxt comes with several built-in optimizations designed to improve your application's performance and contribute to better [Core Web Vitals](https://web.dev/articles/vitals) score. Additionally, there are also multiple plug-in solutions such as modules that will help improve performance and User Experience even more. This guide outlines best practices to optimize performance of your Nuxt application.
+
+## Built-in Solutions
+
+Nuxt offers several built-in solutions that help you optimize performance of your website. Understanding how this works and how you can further customize it to suit your needs is crucial for achieving blazingly fast performance.
+
+### Links
+
+`<NuxtLink>` is a drop-in replacement for both Vue Router's `<RouterLink>` component and HTML's `<a>` tag. It intelligently determines whether the link is internal or external and renders it accordingly with available optimizations (prefetching, default attributes, etc.)
+
+```html
+<template>
+  <NuxtLink to="/about">About page</NuxtLink>
+</template>
+
+<!-- Which will render to with Vue Router & Smart Prefetching -->
+<a href="/about">About page</a>
+```
+
+Nuxt automatically includes smart prefetching. That means it detects when a link is visible (by default), either in the viewport or when scrolling and prefetches the JavaScript for those pages so that they are ready when the user clicks the link.
+
+Read more about it [here](https://nuxt.com/docs/api/components/nuxt-link).
```

Comment:
(and also throughout the rest of the page, we should omit the protocol/host)

```suggestion
Read more about it [here](/docs/api/components/nuxt-link).
```

Reply:
@danielroe you marked this as resolved. Shoukd I change it accordingly to your request or leave it as it it? :)

Reply:
Ah, you have done it already. Nevermind :)